### PR TITLE
Add testing images and update tests

### PR DESCRIPTION
**Pull request by aldenchen**
- Add images to data folder
- Update tests

### CODE REVIEW

**Comment by Sayanti86** [comment]
Looks good to me .

File: test.py [hunk 1]
```diff
@@ -5,71 +5,84 @@
...
+
+    def test_crop_shape(self):
+        '''
+        Function to test that the crop() function returns an image
+        of the correct shape for different input shapes
+        '''
+        for image in test_images:
+
+            cropped_image = crop(image, 15, 10)
+            assert imread(cropped_image).shape == (15, 10, 3)
```
Comment (adityashrm21) [inline test.py]: I think we are not restricting the channels to be equal to 3. The third dimension should not be checked explicitly.
This applies to all the tests in this function except the last one.

File: test.py [hunk 2]
```diff
@@ -125,9 +142,16 @@ def test_compress_size(self):
...
+        
         compressed_img = compress(img_path, 1)
         assert image_size(compressed_img) < 2/8 * image_size(img_path)
 
+        compressed_img = compress(img_path, 6)
+        assert image_size(compressed_img) < 7/8 * image_size(img_path)
+
+        compressed_img = compress(img_path, 8)
+        assert image_size(compressed_img) == image_size(img_path)
+
```
Comment (adityashrm21) [inline test.py:154]: I don't think we should add test cases with such large b values. The tests take a lot of time for compressions with b>5.

File: test.py [hunk 3]
```diff
@@ -113,7 +130,7 @@ def test_compress_value(self):
 
     def test_compress_input_shape(self):
         '''
-        Test that correct image has correct shape
+        Test that correct input image has correct shape
```
Comment (adityashrm21) [inline test.py]: Should be "Test that input image has correct shape".

**Review by adityashrm21 — COMMENTED** [review]
Added some suggested changes. Rest looks good!

File: test.py (hunk 1 above)
Comment (aldenchen) [reply test.py]: Fixed this. You can take a look in the `update_tests` branch.  Will send another pull request soon

File: test.py (hunk 2 above)
Comment (aldenchen) [reply test.py:154]: I think for the tests it should be okay. The test images I created are very small. I just tried it and it's pretty fast with those images.

File: test.py (hunk 3 above)
Comment (aldenchen) [reply test.py]: Fixed. Will send another pull request.

File: test.py (hunk 2 above)
Comment (adityashrm21) [reply test.py:154]: Makes sense!

**Comment by adityashrm21** [comment]
Changes look good now! Merging the PR.

**Pull request merged**